learn/Dockerfile

```diff
@@ -3,12 +3,24 @@ LABEL maintainer='cockpit-devel@lists.fedorahosted.org'
 
 RUN dnf -y update \
  && dnf -y install \
-        lighttpd \
+        nginx \
+        fcgiwrap \
+        spawn-fcgi \
         python3 \
         python3-scikit-learn \
         python3-matplotlib \
- && dnf clean all \
- && mkdir -p /cache/train
+ && dnf clean all
+
+RUN mkdir -m 775 -p /cache/train \
+ && mkdir -m 775 -p /home/user \
+ && mkdir -m 775 -p /cache/client_body_temp \
+ && ln -sf /dev/stdout /var/log/nginx/access.log \
+ && ln -sf /dev/stderr /var/log/nginx/error.log \
+ && rm -rf /usr/share/nginx/html \
+ && ln -snf /cache/images /usr/share/nginx/html \
+ && groupadd -g 1111 -r user \
+ && useradd -r -g user -u 1111 user \
```

Comment:
Can we avoid that for new containers? These won't work for CentOS CI deployments anyway, instead the volumes just need to be world-writable.

Reply:
seems to work fine without, running stuff as the nginx user and making the dirs world-writeable